dexs/tegro/index.ts

```diff
@@ -0,0 +1,95 @@
+import { Adapter } from '../../adapters/types'
+import { gql, request } from 'graphql-request'
+
+const endpoints = {
+  ethereum: 'https://api.thegraph.com/subgraphs/name/ashtegro/tegro',
+  polygon: 'https://api.thegraph.com/subgraphs/name/ashtegro/tegro-polygon',
+  avax: 'https://api.thegraph.com/subgraphs/name/ashtegro/tegro-avax',
+  bsc: 'https://api.thegraph.com/subgraphs/name/ashtegro/tegro-bnb',
+  arbitrum: 'https://api.thegraph.com/subgraphs/name/ashtegro/tegro-arbitrum',
+}
+
+const getNumberOfDay = (date: Date) => {
+  const now: Date = new Date()
+  const timeDifference = now.getTime() - date.getTime()
+  return Math.floor(timeDifference / (24 * 60 * 60 * 1000))
+}
+
+const getVolume = async (queryUrl: string) => {
+  const start = new Date('1970-01-01T00:00:00Z')
```

Comment:
daily volume is`id` dayid should pass from `timestamp`

Reply:
Apologies, it is fixed now. Could you please check?